### PR TITLE
https://issues.redhat.com/browse/ACM-10939 Subm known issue

### PR DESCRIPTION
**Pull request by oafischer**
2.10 and 2.9

### CODE REVIEW

**Review by vthapar — APPROVED** [review]

File: release_notes/known_issues_network.adoc [hunk 1]
```diff
@@ -127,3 +127,32 @@ If you remove a cluster from a `ClusterSet`, or move a cluster to a different `C
 
 You must uninstall Submariner before moving or removing a `ManagedCluster` from a `ManageClusterSet`. If you don't uninstall Submariner, you cannot uninstall or reinstall Submariner anymore and Submariner stops working on your `ManagedCluster`.
 
+[#submariner-install-fail-vmware-ocp]
+=== Submariner installation fails on VMware vSphere with {ocp-short} 4.15 and later
+//2.10:ACM-10939
+
+The Submariner add-on installation fails on VMware vSphere running an {ocp-short} 4.15 and later hub cluster because of a missing broker secret for `ManagedClusters` on the hub cluster. Only the `submariner-addon` pod is created in the `submariner-operator` namespace on `ManagedClusters` and the console shows gateways that are not labeled.
```
Comment (dockerymick) [inline release_notes/known_issues_network.adoc]: is it a `ManagedCluster` resource?

File: release_notes/known_issues_network.adoc [hunk 2]
```diff
@@ -127,3 +127,32 @@ If you remove a cluster from a `ClusterSet`, or move a cluster to a different `C
...
+
+You can work around the issue by creating the secret manually for each `ManagedCluster` in the `ClusterSet` broker namespace. To create the secret manually, complete the following steps:
+
+. Log in to the hub cluster
```
Comment (dockerymick) [inline release_notes/known_issues_network.adoc]:
```suggestion
. Log in to the hub cluster.
```

**Review by dockerymick — CHANGES_REQUESTED** [review]

File: release_notes/known_issues_network.adoc (hunk 1 above)
Comment (vthapar) [reply release_notes/known_issues_network.adoc]: It is a SubmarinerAddon resource created on hub for each ManagedCluster. Not sure if it counts as `ManagedCluster` resource or SubmarinerAddon resource.

Comment (dockerymick) [reply release_notes/known_issues_network.adoc]: @vthapar thanks for confirming! I recommend that "ManagedCluster" not be monospaced and lower-case (managed cluster). @oafischer can you explain the intention for monospacing managed cluster that I might not be aware of?

Comment (oafischer) [reply release_notes/known_issues_network.adoc]: @dockerymick There is no additional reason for monospacing it. I will remove the monospace, thanks for the review!

**Pull request merged**